changelog/unreleased/issue-3334

```diff
@@ -0,0 +1,7 @@
+Bugfix: `created new cache in' message is only printed on a terminal
+
+`created new cache` message was outputed even when the output wasn't a
+terminal. That broke piping `restic dump` output to tar or zip
+if cache directory didn't exist.
+
+https://github.com/restic/restic/issues/3334
```

Comment:
Please also include a link to this PR:
`https://github.com/restic/restic/pull/3343`